src/ledger/ledger.ts

```diff
@@ -10,7 +10,32 @@ export const lndAccountingPath = "Assets:Reserve:Lightning" // TODO: rename to A
 export const escrowAccountingPath = "Assets:Reserve:Escrow" // TODO: rename to Assets:Lnd:Escrow
 
 // liabilities
-export const accountPath = (uid) => `Liabilities:${uid}`
+export const liabilitiesMainAccount = "Liabilities"
+export const accountPath = (uid) => `${liabilitiesMainAccount}:${uid}`
+export const resolveAccountId = (accountPath: string | string[]) => {
```

Comment:
is this still needed at all?

I don't think we typically want to merge `accountPath` and `accounts` (I think we should eventually have just one of those 2 options in our codebase to avoid the issue)

Reply:
I prefer this instead of use index in account path, also it is safer in case of account path changes (it is unit tested)

Reply:
but I'm still not sure to understand why we want the overhead of parsing both a table and a string. this could lead to us mixing types which would be bad IMO

Reply:
this is because of medici, account paths are strings (accounts) or arrays of strings (account_path)